packages/core/tailwind.config.ts

```diff
@@ -1,5 +1,5 @@
 import type { Config } from 'tailwindcss';
-import { theme } from './theme';
+import { theme } from './src/theme';
```

Comment:
**issue:** `theme.ts` is references in `package.json::exports` - if we rename it, we need to fix that field, too

Reply:
Ah good catch. For reference: did this to avoid any issues involving vite's filesystem access scoping